exist-versioning-release.md

```diff
@@ -150,48 +231,72 @@ You will require a system with:
             <!-- eXist-db Github Release -->
             <server>
                 <id>github</id>
-                <privateKey>[Github Personal access tokens]</privateKey>
+                <privateKey>YOUR-Github-ACCESS-TOKEN</privateKey>
             </server>
         </servers>
+        
+        <profiles>
+           <profile>
+               <id>existdb-release-signing</id>
+               <properties>                   
+                      
+                   <existdb.release.key>YOUR-GPG-KEY-ID</existdb.release.key>
+                   <existdb.release.public-keyfile>/your/part/to/.gnupg/pubring.gpg</existdb.release.public-keyfile>
+                   <existdb.release.private-keyfile>/your/part/to/.gnupg/secring.gpg</existdb.release.private-keyfile>
+                   <existdb.release.key.passphrase>YOUR-GPG-PASSWORD</existdb.release.key.passphrase>
+
+                   <existdb.release.keystore>/your/path/to/.keystore</existdb.release.keystore>
+                   <existdb.release.keystore.pass>YOUR-KEYSTORE-PASSWORD</existdb.release.keystore.pass>
+                   <existdb.release.keystore.key.alias>YOUR-KEYSTORE-ALIAS</existdb.release.keystore.key.alias>
+                   <existdb.release.keystore.key.pass>YOUR-KEYSTORE-PASSWORD</existdb.release.keystore.key.pass>
+               </properties>
+           </profile>
+        </profiles>
+        <activeProfiles>
+            <activeProfile>existdb-release-signing</activeProfile>
+        </activeProfiles>
     </settings>
+```
+
+1. Merge any outstanding PRs that have been reviewed and accepted for the milestone eXist-5.3.0.
+
+2. Create a new clone of the branch you want to release from into a new folder.
+   In the example below the `develop` branch is cloned into the folder `release`
+
+
+    ```bash
+    git clone \ 
+      -b develop \
```

Comment:
Instead of `-b developer` consider the faster options `--single-branch --branch develop`

Reply:
I did use `--single-branch` and yes that is much faster. Do you happen to know if the later commands will still work? Especially `git checkout -b next-master eXist-5.3.0`. If that is the case I will re-add that.
Do you also want to say that long parameter names are preferred (`-b` is `--branch`)

Reply:
I think the `next-master` bit is superfluous - I just added a comment about that.

>Do you also want to say that long parameter names are preferred

I like to see long names in tutorials because then the parameter is not opaque, i.e. the reader can pretty much understand what it means without having to find the manual page and lookup `-b` to see what it stands for.